### ISSUE TITLE
Check if iOS 13.2.2 has bugs in SwiftUI because it misbehaved in iPhone 11 Pro Max

### ISSUE THREAD

**Issue by DmitryBespalov** [gnosis/safe-ios#160]

**Comment by DmitryBespalov** [comment]
Resolution: the workarounds for bugs on version 13.2.2 are breaking the workarounds for version 13.3/4/5, I am for disabling the support for 13.2.2 in favor of 13.3 and higher.

Issues:
- App crashing when navigating back from the Load Safe screen or edit safe name screen
- Enter Safe Address screen is shown under the navigation bar.

The root cause of all these problems is the custom navigation bar that we have to mimic instead of using the system navigation bar. We already spent ~week of effort on implementing it and finding a way for it to work, with workarounds of the bugs of the iOS, which does not expect us to use it in that way that we have. Because of custom navigation bar, we have to make different hacks, which led to issues on iOS 13.2.2 on the iPhone 11 Pro Max device.

**Comment by DmitryBespalov** [comment]
iOS 13.3 fixes the crash, but introduces another issue: the buttons opening the screens work only once, unless we do a hack/workaround for that.
